src/Mvc/Mvc.Core/src/JsonOptions.cs

```diff
@@ -26,6 +27,12 @@ public class JsonOptions
 
             // Use camel casing for properties
             PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+
+            Converters =
```

Comment:
I think it's a much better idea to put these converters on the types themselves (attributes) than our settings object. There's no use-case for overriding the converter, and we want it to work trivially without our settings object.

Reply:
Huh, I was under the impression that adding the attribute would cause it to appear in the ref assembly and it'd be a hassle to remove it in the future. But appears not. I'll update

Reply:
Done

Reply:
Even if that's the cost then it's fine right? Like ultimately if we have to have a converter for these types its fine.

Reply:
Ultimately we want to switch to using https://github.com/dotnet/corefx/issues/39416. I really don't want us to maintain these converters long term

Reply:
I don't necessarily understand it. Using a converter is the power tool, it's not bad it's just more expensive to write the first time, and we already have to do it.